docs/source/configuration/refine_admin.rst

```diff
@@ -0,0 +1,35 @@
+Refine Admin
+============
+
+Some administrative functionality is built out using `Refine <https://refine.dev>`_. This is a separate service and requires a bit of setup to make it work with the MITxOnline app. 
+
+Accessing
+---------
+
+The Docker compose environment builds a ``refine`` container that runs a dev server available at port 8016. 
+
+Set Up
+------
+
+The Refine app uses OAuth2/OIDC for authentication with the core MITxOnline app. You will need to generate a shared private key for this authentication flow to work. 
+
+*Generating the Key:* The following command will generate the key:
+
+``openssl genrsa -out oidc.key 4096``
+
+The key then needs to be put into the ``.env`` file into a setting called ``OIDC_RSA_PRIVATE_KEY``. Remove linebreaks and replace with the string ``\n``. Both the Refine and Django apps will pick the key up from this setting.
```

Comment:
Note to add: You need to restart the server after add the key in .env file

Reply:
I noted, you added the note below but It should be here regarding restarting container as when I was adding the below confituration in `Application` model then showing error to have `OIDC_RSA_PRIVATE_KEY`.

Reply:
I updated this a bit - there's a one-liner in there now that will output the key in the right format directly to the terminal, so you won't have to deal with key files at all (which removes the possibility that the key gets committed to the repo accidentally too).